jerry-core/parser/js/js-lexer.c

```diff
@@ -971,6 +979,12 @@ lexer_parse_number (parser_context_t *context_p) /**< context */
   context_p->source_p = source_p;
 } /* lexer_parse_number */
 
+/**
+ * One character long token (e.g. comma).
+ *
+ * @param char1 character
```

Comment:
Do we use at-param with macro functions?

Reply:
Yes, we do. There are a few examples in the current master. Unfortunately we cannot use inline comments for macro functions parameters.

Reply:
Few examples does not make a rule. I found far less positive examples than negative, so I would rather remove these.

Reply:
Doxygen will warn if I remove these.

Reply:
OK